store_to_db.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/local/bin/python3
+""" Module to gzip and store a directory in the database """
+import os
+import datetime
+import time
+import json
+import re
+import argparse
+import tempfile
+import mysql.connector
+
+
+def exec_fetchone(cursor, sql_stmt, params=None):
+    """executes the sql stmt and fetches the first one in the result list"""
+    cursor.execute(sql_stmt, params)
+    result = cursor.fetchone()
+    return result[0]
+
+
+def connect_to_db(database):
```

Comment:
I think we already have implementations of `connect_to_db` and `write_file_to_db` elsewhere in this repository. Let's refactor them into a common module rather than re-writing them.

Reply:
sure.